test/TestUtilities/Test.Utility/PackageManagement/TestSolutionManager.cs

```diff
@@ -172,6 +172,7 @@ public void EnsureSolutionIsLoaded()
 
         public Task<bool> DoesNuGetSupportsAnyProjectAsync()
         {
+            // NOTE cannot use TaskResult.True here because of IVT diamond. This is only test code.
```

Comment:
Please spell out whatever IVT diamond is/means? Otherwise, this comment just means the reader has to look it up to get on the same page. 
```suggestion
            // NOTE: cannot use `TaskResult.True` here because of _______.
```

Reply:
IVT == InternalsVisibleTo.

Diamond == https://en.wikipedia.org/wiki/Multiple_inheritance#The_diamond_problem

I've removed these comments as they don't seem to add much, and again this is just test code. There are a bunch of other places in test assemblies where I couldn't do this for the same reason, so it's not that helpful to call out these two particular instances.

Reply:
Ah ok, sure that's fine. Thanks for defining it for me, though :)